### PR TITLE
Rating

### PR DESCRIPTION
**Pull request by arpit01923**
Done with a Rating component
image

### CODE REVIEW

**Review by anupkgurung — APPROVED** [review]
looks good.

File: css/example/Rating.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../components/Rating.css">
+    <script src="https://kit.fontawesome.com/dcbd873a92.js" crossorigin="anonymous"></script>
```
Comment (ianuragsharma) [inline css/example/Rating.html]: Rather than using a script for fontawsome,you can use cdn link in your style sheet

**Review by ianuragsharma — APPROVED** [review]
LGTM

File: css/example/Rating.html (hunk 1 above)
Comment (arpit01923) [reply css/example/Rating.html]: done

**Pull request merged**